python/SWESimulators/CDKLM16.py

```diff
@@ -280,75 +283,75 @@ def step(self, t_end=0.0, apply_stochastic_term=True):
                     wind_speed=self.wind_stress.wind_speed, \
                     wind_direction=self.wind_stress.wind_direction + perturbation)
                 # Upload new wind stress params to device
-                cl.enqueue_copy(self.cl_queue, self.wind_stress_dev, new_wind_stress.tostruct())
+                #cl.enqueue_copy(self.cl_queue, self.wind_stress_dev, new_wind_stress.tostruct()) # FIXME! CL->CUDA
```

Comment:
Should this fixme be here, or should it be fixed?

Reply:
Will fix.